libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/ImageUtils.java

```diff
@@ -428,8 +428,12 @@ private static boolean resizeImageAndWriteToStream(Context context,
         // determine correct scale value (should be power of 2)
         // http://stackoverflow.com/questions/477572/android-strange-out-of-memory-issue/3549021#3549021
         int scale = 1;
-        if (maxWidth > 0 && optBounds.outWidth > maxWidth) {
-            double d = Math.pow(2, (int) Math.round(Math.log(maxWidth / (double) optBounds.outWidth) / Math.log(0.5)));
+        boolean isPortrait = optBounds.outWidth < optBounds.outHeight;
+        int optBoundsMaxSize = Math.max(optBounds.outWidth, optBounds.outHeight);
```

Comment:
Can we declare here `double largestSizeOfOutBounds = Math.max(optBounds.outWidth, optBounds.outHeight);` and get rid of both `isPortrait` and `optBoundsMaxSize` ?